src/host/screenInfo.cpp

```diff
@@ -1972,6 +1981,16 @@ void SCREEN_INFORMATION::UseMainScreenBuffer()
             psiMain->ProcessResizeWindow(&(psiMain->_rcAltSavedClientNew), &(psiMain->_rcAltSavedClientOld));
             psiMain->_fAltWindowChanged = false;
         }
+
+        // GH#381: When we switch into the alt buffer:

```

Comment:
Assumedly this should be when we switch in the **main** buffer.

Reply:
you caught me copy pasting :P